chips/sam4l/src/usart.rs

```diff
@@ -773,11 +818,13 @@ impl hil::spi::SpiMaster for USART {
 
     /// Pass in a None to use the HW chip select pin on the USART (RTS).
     fn specify_chip_select(&self, cs: Self::ChipSelect) {
+        self.enable_clock();
```

Comment:
Right. But setting the chip select doesn't actually touch registers, does it? It's just a struct member